### DISCUSSION TITLE
Signature thumbprint do not match expected - Error in windows image

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [actions/virtual-environments#9892] [General questions] [answered]
Hi Team,

we are getting the error "Signature thumbprint do not match expected" for several softwares while installing for windows.

Please find the logs of the errors below.
![image](https://github.com/actions/runner-images/assets/113025693/74383593-b0f6-4785-930f-f33e4bb32347)

![image](https://github.com/actions/runner-images/assets/113025693/c22f1394-8ed1-4bc0-a309-b3c6a4487360)

Can you please check and solve the same. We are using the image Ubuntu 22.04

please let me know if you need more info.

i look forward for your response.

**Comment by erik-bershel** [comment, marked as answer]
All of those fixed by now. Please update your fork to be in consistency with `main` branch.

**Comment by LokeshNagireddy** [comment]
Hi @erik-bershel , Thanks for the solution, but i got a new error in the build after 6hours of run. Please find the details below.

![image](https://github.com/actions/runner-images/assets/113025693/2f5afa4b-71d5-4df8-b3c6-8b59ce1ca140)

Can you please check and solve the issue.

i look forward for your response.

**Comment by LokeshNagireddy** [comment]
Issue solved
